app/AndroidManifest.xml

```diff
@@ -357,10 +371,18 @@
         <receiver android:name="org.commcare.provider.ExternalApiReceiver">
             <intent-filter>
                 <action android:name="org.commcare.dalvik.api.action.ExternalAction"/>
+                <category android:name="android.intent.category.DEFAULT"/>
+            </intent-filter>
+        </receiver>
 
+        <receiver android:name="org.commcare.provider.CommCareLogoutReceiver"
+            android:permission="org.commcare.dalvik.permission.COMMCARE_LOGOUT">
+            <intent-filter>
+                <action android:name="org.commcare.dalvik.api.action.CommCareLogoutAction"/>
                 <category android:name="android.intent.category.DEFAULT"/>
             </intent-filter>
         </receiver>
+
         <receiver android:name="org.commcare.provider.RefreshToLatestBuildReceiver">
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/8796bd65a3cac92cc7a4aee39619346d04998703/app/AndroidManifest.xml#L386"></span>Exported receiver does not require permission</td>
    </tr>
  </tbody>
</table>